packages/cacti-plugin-copm-fabric/Makefile

```diff
@@ -0,0 +1,110 @@
+WORKSPACE ?= $(shell realpath `pwd`/../../)
+DOCKER_IMAGES=hyperledger/fabric-orderer hyperledger/fabric-peer hyperledger/fabric-ca ghcr.io/hyperledger/cacti-weaver-iin-agent ghcr.io/hyperledger/cacti-weaver-relay-server ghcr.io/hyperledger/cacti-weaver-driver-fabric
```

Comment:
I missed this earlier. Thanks to @sandeepnRES for noticing. The project is now in the `hyperledger-cacti` organization, so please rename `ghcr.io/hyperledger/...` to `ghcr.io/hyperledger-cacti/...`.

Reply:
The intention of this line of the makefile is to clean existing docker images.  As of this morning, the docker image referenced have not been moved to the new repository, so changing this line will make the code not work.

Reply:
Right. We'll try to submit a release PR next week to publish packages to `hyperledger-cacti`. We haven't done this yet because we are still figuring out the best way to update the scripts.

Can you please open an issue to track this? Assign it to me. As soon as the packages are released, we ought to change these references.

Reply:
This has been opened here: https://github.com/hyperledger-cacti/cacti/issues/3609